### PR TITLE
Update to v95

### PR DESCRIPTION
**Pull request by johnegarza**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -23,13 +23,15 @@ WORKDIR /opt/lib/perl/VEP/Plugins
 
 RUN wget https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/93/Downstream.pm \
```
Comment (tmooney) [inline Dockerfile]: Should this one also get updated to 95?
```suggestion
RUN wget https://raw.githubusercontent.com/Ensembl/VEP_plugins/release/95/Downstream.pm \
```

**Comment by apaul7** [comment]
dbNSFP.pm
REVEL.pm
CADD.pm
All have changes in 95 compared to 93. While the plugins should continue to work in the 93 version we should try to be consistent in keeping them at the version of VEP we're using.

**Review by tmooney — APPROVED** [review]
+1 🌃

**Comment by tmooney** [comment]
(As a side note, since we like to build branches as Docker tags, I'd advocate for making PRs from a fork against master (to generate `latest`) and then branching for the version afterwards.  (Or possibly making a PR against the version branch for subsequent bugfixes if master has moved on.)

**Review by jasonwalker80 — APPROVED** [review]
+1, as-is this looks good to me. However, #5 was created to address the version/image name.

**Review by dufeiyu — APPROVED** [review]
+1 Looks good to me.

**Pull request merged**